static/main.css

```diff
@@ -234,69 +220,100 @@ textarea:focus {
     opacity: 0;
   }
 }
-footer {
+.Footer {
   background-color: #c49542;
-  color: #231f20;
-  padding-bottom: 50px;
-  padding-top: 50px;
+  font-family: Roboto;
+  padding-bottom: 3rem;
+  padding-top: 3rem;
   text-align: center;
 }
-.fTitle {
+.Footer >div:first-child {
   color: #fff;
-/* This is not working, and I'm not sure why */
   font-family: Arvo;
-/* This is not working, and I'm not sure why */
   font-weight: bold;
+  text-transform: uppercase;
 }
-.fInfo {
-/* This is not working, and I'm not sure why */
-  font-family: Roboto;
-/* This is not working, and I'm not sure why */
+.Footer a {
+  color: #000;
 }
-.fLink,
-.fLink:hover {
-  color: #231f20;
-/* This is not working, and I'm not sure why */
-  font-family: Roboto;
-/* This is not working, and I'm not sure why */
-  text-decoration: none;
+.Header {
+  display: flex;
 }
-.hDiv {
+.Header div {
   display: inline-block;
-  padding-top: 25px;
+  flex: 1;
+  font-family: Roboto;
+  padding-top: 2rem;
   text-align: center;
-  width: 19.7%;
+  text-transform: uppercase;
 }
-.hLink {
-  color: #231f20;
-/* This is not working, and I'm not sure why */
-  font-family: Roboto;
-/* This is not working, and I'm not sure why */
+.Header div.selected {
+  color: #c49542;
 }
-.hCurrent {
+.Header div:nth-child(3) {
   color: #c49542;
-/* This is not working, and I'm not sure why */
   font-family: Arvo;
-/* This is not working, and I'm not sure why */
   font-weight: bold;
 }
-.hLogo {
-  padding-top: 40px;
-  width: 100%;
+.Header div:hover {
+  text-decoration: underline;
 }
-.hImg {
-  height: 150px;
+div img {
+  height: 15rem;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 2rem;
 }
-.homePage {
+div div {
+  padding-top: 2rem;
   text-align: center;
-  padding-top: 30px;
 }
-.homeTitle {
+div >div {
+  font-family: Arvo;
+  font-size: 2rem;
+}
+.mHeadline {
+  font-family: Arvo;
   font-size: 30px;
+  padding-top: 30px;
+  text-align: center;
+}
+.mSubtitle {
+  font-family: Roboto;
+  text-align: center;
 }
-.homeParagraph {
+.mParagraph {
+  font-family: Roboto;
+  margin-left: 30%;
+  margin-right: 30%;
+  text-align: center;
   width: 40%;
 }
+.mRow {
```

Comment:
don't use `m` prefixes, just let the hierarchy of class names give context to the class name